benchmarks/distributed/lambda_runtimes/test_lambda.py

```diff
@@ -0,0 +1,122 @@
+import boto3
+
+import json
+import time
+import cloudpickle
+import base64
+import math
+
+
+NUM_RUNS = 4
```

Comment:
Can we add some comments on how to run/reproduce this benchmark?

That'd be helpful. Ideally, I'd like to run this myself. But right now, I have no clue how to start.